src/content/docs/zh-cn/start/frontend/vite.mdx

```diff
@@ -101,7 +101,7 @@ Vite 是为提供更快速，更简洁的现代 Web 项目开发体验而诞生
         host: host || false,
         port: 5173,
       },
-      // 为了使用命令行接口(CLI)为当前构建目标配置的 Tauri 环境变量
+      // 添加有关当前构建目标的额外前缀，以访问命令行接口 (CLI) 设置的 Tauri 环境变量 
```

Comment:
@RSS1102 does this sounds right for you? It sounds contrived. Can you help me on this?

Reply:
😀 yeah, It seems that the translation has surpassed the original work(I seem to think he has more truth, maybe I have a misunderstanding of the original meaning? In a discord discussion if you can.).